linkcheck/ftpparse.py

```diff
@@ -78,20 +92,20 @@ def ftpparse(line):
         parts = line.split()
         if len(parts) < 7:
             return None
-        del parts[0] # skip permissions
+        del parts[0]  # skip permissions
         if parts[0] != 'folder':
-            del parts[0] # skip nlink
-        del parts[0] # skip uid
-        del parts[0] # skip gid or size
+            del parts[0]  # skip nlink
+        del parts[0]  # skip uid
+        del parts[0]  # skip gid or size
         if not ismonth(parts[0]):
-            del parts[0] # skip size
+            del parts[0]  # skip size
         if not ismonth(parts[0]):
             return None
-        del parts[0] # skip month
-        del parts[0] # skip day
+        del parts[0]  # skip month
+        del parts[0]  # skip day
         if not parts:
             return None
-        del parts[0] # skip year or time
+        del parts[0]  # skip year or time
         name = " ".join(parts)
```

Comment:
Wow that is interesting code.  It's trying to handle filenames with spaces?  I don't think it'll handle filenames with multiple consecutive spaces correctly.

(Yes, out of scope for this PR, but you can't put code in front of me and not have me read it!)